src/Interface/Application/ModuleWidget.cc

```diff
@@ -320,17 +325,20 @@ QGroupBox* ModuleWidgetDisplay::getButtonGroup() const
 static const int UNSET = -1;
 static const int SELECTED = -50;
 
-typedef boost::bimap<QString, int> ColorStateLookup;
-typedef ColorStateLookup::value_type ColorStatePair;
-static ColorStateLookup colorStateLookup;
-void fillColorStateLookup(const QString& background);
-
 namespace
 {
   ModuleId id(ModuleHandle mod)
   {
     return mod ? mod->id() : ModuleId();
   }
+
+  QString backgroundColorByName(const QString& name)
+  {
+    if (name.contains("Loop"))
+      return moduleRGBA(13, 152, 186);
+
+    return moduleRGBA(99,99,104);
```

Comment:
Same with these numbers.

Reply:
Do color RGB parameters need names everywhere? I'm all against magic numbers in general, but it would be overkill to have to write this: `return moduleRGBA(NOT_MUCH_RED, SOME_GREEN, LOTS_OF_BLUE)`

Reply:
I think I was too deep into looking at this. The function clearly say RGBA in it so I think the numbers are self explanatory in this case. It would be overkill.